Add additional help when AttributeError is raised with regards check_tty

edwardbarak opened the pull request:
In reference to issue #52 I thought it might be useful to include an error message for people to try setting check_tty=False so this question doesn't have to be re-answered.

verigak commented:
Thanks, can you reformat this to pass pycodestyle and squash the commits into one?

edwardbarak commented:
I squashed the commits. The only thing that came up when I ran it through pycodestyle linter was a missing docstring for the function, but I don't actually know enough about the function to write a docstring for it. Just enough to fix the error that I fixed.

verigak commented:
Thanks!
